### DISCUSSION TITLE
How to enable QuickView tab

### DISCUSSION THREAD

**Discussion by MatteoBiasi** [noi-techpark/it.bz.opendatahub.databrowser#240] [Q&A]
Hi @gappc, we were not able to understand which is the logic behind the visibility of the QuiView tab button. How can we make it visible? Thanks!

**Comment by gappc** [comment]
Hi @MatteoBiasi,

the logic when to show the Quick View button can be found in the [datasetConfigStore.ts](https://github.com/noi-techpark/it.bz.opendatahub.databrowser/blob/development/databrowser/src/domain/datasetConfig/store/datasetConfigStore.ts) file, which is a [pinia](https://pinia.vuejs.org/) store.

It provides getters to check if the current configuration defines a Quick View (see `hasQuickView`) and if the current route is a Quick View (see `isQuickView`).

The `hasQuickView` getter is used in the [DatasetNavigation.vue](https://github.com/noi-techpark/it.bz.opendatahub.databrowser/blob/development/databrowser/src/domain/datasets/header/DatasetNavigation.vue) and [TableView.vue](https://github.com/noi-techpark/it.bz.opendatahub.databrowser/blob/development/databrowser/src/domain/datasets/tableView/TableView.vue) components to decide if buttons / tab links to Quick View should be visible.

> For example, the view configuration for `Article` is defined inside [article.views.ts](https://github.com/noi-techpark/it.bz.opendatahub.databrowser/blob/development/databrowser/src/config/tourism/article/article.views.ts). If you add the missing property `quick` you should start to see the buttons / tabs to the Quick View

The main implementation of the Quick View itself starts in the [QuickView.vue](https://github.com/noi-techpark/it.bz.opendatahub.databrowser/blob/development/databrowser/src/domain/datasets/quickView/QuickView.vue) component.

I hope those directions help you getting started. If you have any questions, don't hesitate to ask.
